tree/treeplayer/src/TCsvDS.cxx

```diff
@@ -393,24 +436,11 @@ void TCsvDS::SetNSlots(unsigned int nSlots)
 
 void TCsvDS::Initialise()
 {
-   const auto nRecords = fRecords.size();
-   const auto chunkSize = nRecords / fNSlots;
-   const auto remainder = 1U == fNSlots ? 0 : nRecords % fNSlots;
-   auto start = 0UL;
-   auto end = 0UL;
-
-   for (auto i : ROOT::TSeqU(fNSlots)) {
-      start = end;
-      end += chunkSize;
-      fEntryRanges.emplace_back(start, end);
-      (void)i;
-   }
-   fEntryRanges.back().second += remainder;
 }
```

Comment:
we can just remove `Initialise` from `TCsvDS` right? (`TDataSource` implements an empty one)

Reply:
Removed